Cesium.CodeGen/Ir/Types/CTypeSystem.cs

```diff
@@ -16,4 +16,6 @@ internal class CTypeSystem
     public IType CharPtr { get; } = new PrimitiveType(PrimitiveTypeKind.Char).MakePointerType();
     public IType Float { get; } = new PrimitiveType(PrimitiveTypeKind.Float);
     public IType Double { get; } = new PrimitiveType(PrimitiveTypeKind.Double);
+
+    public int GetTypeSize(IType type) => type.SizeInBytes;
```

Comment:
Why do we need this? So now there's two ways of achieving the same.

Reply:
I would like to get rid of `SizeInBytes` as away to get type size. My opinion that Type knows too much about thinks, but maybe you are right here. When I prepare answer I probably found answer to #132

Reply:
I don't think that's an answer to #132, unfortunately, but that's an interesting thought anyway.